apps/spruce/src/components/Redirects/ImageRedirect.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate } from "react-router-dom";
+import { getImageRoute, ImageTabRoutes } from "constants/routes";
+import { useFirstImage } from "hooks";
+
+export const ImageRedirect: React.FC = () => {
```

Comment:
I don't think this redirect does anything at the moment; you'd have to add it to` Content/index.tsx` for it to have an effect

Reply:
Forgot to remove it before - done!